dist/.goreleaser.yaml

```diff
@@ -63,22 +65,22 @@ builds:
       - -X github.com/scylladb/scylla-manager/v3/pkg.version={{ .Version }}
 
 archives:
-  - name_template: "{{ .ProjectName }}_{{ .Version }}.{{ .Os }}_{{ .Arch }}"
+  - name_template: >-
+      {{ .ProjectName }}_{{ .Version }}.{{ .Os }}_{{- if eq .Arch "amd64" }}x86_64{{- else if eq .Arch "arm64" }}aarch64{{ end }}
     files:
       - etc
       - license/LICENSE*
       - scripts
       - systemd
-    replacements:
-      amd64: x86_64
-      arm64: aarch64
+    rlcp: true
 
 nfpms:
   - id: server
     builds:
       - server
     package_name: "scylla-manager-server"
-    file_name_template: "{{ .ProjectName }}-server-{{ .Version }}_{{ .Os }}.{{ .Arch }}"
+    file_name_template: >-
+      {{ .ProjectName }}_{{ .Version }}.{{ .Os }}_{{- if eq .Arch "amd64" }}x86_64{{- else if eq .Arch "arm64" }}aarch64{{ end }}
```

Comment:
Should the new file_name_template include `-server-` ? It's missing in the new template.
The same for client and agent.

Reply:
fixed